basic_clock/main.py

```diff
@@ -0,0 +1,46 @@
+"""An NTP time app"""
+___name___         = "NTP time"
+___license___      = "MIT"
+___dependencies___ = ["ntp", "wifi", "app"]
+___categories___   = ["EMF"]
+
+# borrowed from https://github.com/micropython/micropython/blob/master/esp8266/scripts/ntptime.py
+
+import ugfx, ntp, wifi, utime, machine, app
+from tilda import Buttons
+# initialize screen
+ugfx.init()
+ugfx.clear()
```

Comment:
you will have to set the backlight pin to high as well, otherwise this won't work if it's the default app.

Reply:
but then again, chances are something else will set it first.